docs/source/en/glossary.mdx

```diff
@@ -93,11 +85,6 @@ The backbone is the network (embeddings and layers) that outputs the raw hidden
 
 Color images are made up of some combination of values in three channels - red, green, and blue (RGB) - and grayscale images only have one channel. In 🤗 Transformers, the channel can be the first or last dimension of an image's tensor: [`n_channels`, `height`, `width`] or [`height`, `width`, `n_channels`].
 
-### causal language modeling
-
-A pretraining task where the model reads the texts in order and has to predict the next word. It's usually done by
-reading the whole sentence but using a mask inside the model to hide the future tokens at a certain timestep.
```

Comment:
This definitely need to be kept, we use it all the time in the course and the examples. The class is `AutoModelForCausalLM` etc.